src/index.html

```diff
@@ -8,6 +8,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="main">
+      <div class="card" data-qa="card">
+        <img
+          alt = "APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA/A)"
+          class = "card__image"
+          src="images/imac.jpeg"
+        >
+        <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__ratings">
+          <ul class="card__stars-4">
```

Comment:
Consider using classes from the previous task, the modifier change should work for this block